src/kakuro/GameController.java

```diff
@@ -210,9 +212,45 @@ public Boolean solveBoard()
     	  return true;
           else
           return false;
-    	
-    	
-    	
-    	
+    }
```

Comment:
think we need to disregard the moving out part as Seva already merged his and I merged it to the epic branch